src/lib.rs

```diff
@@ -172,3 +174,42 @@ impl<T: BrombergHashable> BrombergHashable for alloc::rc::Rc<T> {
         (**self).bromberg_hash()
     }
 }
+
+impl Digest for HashMatrix {
```

Comment:
I notice that the digest crate has some additional traits, which imply an instance of Digest. I haven't looked too far into it; any thoughts on whether it's better to implement those traits, vs implementing this one directly? If nothing else, I'd expect that implementing those would give a bit more granularity for downstream users.

Reply:
You're right, I naively assumed they would automatically get implemented using this `Digest` trait, but it looks like it's the other way around: the `Digest` trait gets automatically implemented when the impl for the other traits exist.

I'll make that change tomorrow 😅